### PR TITLE
Add bigwig to dnaseq docker

### PR DESCRIPTION
**Pull request by JenkoB**
## Checklist

* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [ ] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [ ] All inputs are used in process.
* [ ] All output fields have their ``re-save`` calls.

### CODE REVIEW

**Comment by JenkoB** [comment]
@mzganec @JureZmrzlikar please review

File: resolwe_bio/docker_images/dnaseq/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Included bioinformatics tools:
 * CheMut
 * CNVkit (0.9.3)
 * GATK (4.0.2.1)
+* deepTools (3.1.0)
```
Comment (mzganec) [inline resolwe_bio/docker_images/dnaseq/README.md]: Order alphabetically.

File: resolwe_bio/docker_images/dnaseq/packages-debian.txt [hunk 2]
```diff
@@ -6,3 +6,4 @@ python3-setuptools
 python3-wheel
 r-base-core
 tabix
+libpython3.6-dev
```
Comment (mzganec) [inline resolwe_bio/docker_images/dnaseq/packages-debian.txt]: Order alphabetically.

File: resolwe_bio/docker_images/dnaseq/packages-python3.txt [hunk 3]
```diff
@@ -3,6 +3,7 @@ bokeh==0.12.13 --hash=sha256:6e28cbfd88de0c459435278b75f9982591ec0aaa3d37a195052
 pandas==0.22.0 --hash=sha256:c372db80a5bcb143c9cb254d50f902772c3b093a4f965275197ec2d2184b1e61
 matplotlib==2.1.2 --hash=sha256:2452beef35840bdd9c3f613fc112ce6eb449cabca2bcb8a3f0d7eedd8e11d85d
 cnvkit==0.9.3 --hash=sha256:a10ee65b946c0708b2ee7f40f291fcd4a6af3811f0cc77791ea723b66e306f16
+deepTools==3.1.0 --hash=sha256:cba820f4e8525ef3ea1e1a7b7b5c621ac1b27627882a2d0c9aa502ca2cb9a405
```
Comment (mzganec) [inline resolwe_bio/docker_images/dnaseq/packages-python3.txt]: Order alphabetically.

File: resolwe_bio/docker_images/dnaseq/packages-python3.txt [hunk 4]
```diff
@@ -23,3 +24,34 @@ future==0.16.0 --hash=sha256:e39ced1ab767b5936646cedba8bcce582398233d6a627067d4c
...
+plotly==2.7.0 --hash=sha256:f7305816a423fd899da20919282445703118aeece19a54fb97be4cbf302341f8
+py2bit===0.3.0 --hash=sha256:450555c40cba66957ac8c9a4b6afb625fb34c4bb41638de78c87661ff8b682ef
+pyBigWig==0.3.11 --hash=sha256:408ebb40f01c72c77adde4d785a18dabc9abbe9020024e4296b8f6a51a662ae7
+Pygments==2.2.0 --hash=sha256:78f3f434bcc5d6ee09020f92ba487f95ba50f1e3ef83ae96b9d5ffa1bab25c5d
+pyparsing==2.2.0 --hash=sha256:fee43f17a9c4087e7ed1605bd6df994c6173c1e977d7ade7b651292fab2bd010
+sphinx==1.7.5 --hash=sha256:85f7e32c8ef07f4ba5aeca728e0f7717bef0789fba8458b8d9c5c294cad134f3
+sphinxcontrib-websupport==1.1.0 --hash=sha256:68ca7ff70785cbe1e7bccc71a48b5b6d965d79ca50629606c7861a21b206d9dd
+snowballstemmer==1.2.1 --hash=sha256:9f3bcd3c401c3e862ec0ebe6d2c069ebc012ce142cce209c098ccb5b09136e89
+traitlets==4.3.2 --hash=sha256:c6cb5e6f57c5a9bdaa40fa71ce7b4af30298fbab9ece9815b5d995ab6217c7d9
+requests==2.18.4 --hash=sha256:6a1b267aa90cac58ac3a765d067950e7dbbf75b1da07e895d1f594193a40a38b
```
Comment (mzganec) [inline resolwe_bio/docker_images/dnaseq/packages-python3.txt]: Order alphabetically.

**Comment by JenkoB** [comment]
@JureZmrzlikar @jkokosar please review

**Pull request merged**